tests/test_holiday_groups.py

```diff
@@ -37,3 +38,27 @@ def _populate(self, year):
         test_holidays._add_christmas_day_three("Third day")
         self.assertIn("2022-12-27", test_holidays)
         self.assertEqual(1, len(test_holidays))
+
+
+class TestInternationalHolidays(TestCase):
+    def test_add_childrens_day(self):
+        class TestHolidays(HolidayBase, InternationalHolidays):
+            def __init__(self, *args, **kwargs):
+                InternationalHolidays.__init__(self)
+                super().__init__(*args, **kwargs)
+
+            def _populate(self, year):
+                super()._populate(year)
+
+        test_holidays = TestHolidays()
+
+        test_holidays._populate(2022)
+        test_holidays._add_childrens_day("Children's Day (November 20)", NOV)
```

Comment:
Just a note: using "NOV" instead of NOV would make NOV importing unnecessary with pretty much the same readability.

Reply:
That works for me - I will switch to that version then 🙏